azure-pipelines.yml

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) 2020 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+# Imports the shared Azure CI tools
+resources:
+  repositories:
+    - repository: templates
+      type: github
+      name: shotgunsoftware/tk-ci-tools
+      ref: refs/heads/master
+      endpoint: shotgunsoftware
+
+# We want builds to trigger for 3 reasons:
+# - The master branch sees new commits
+# - Each PR should get rebuilt when commits are added to it.
+# - When we tag something
+trigger:
+  branches:
+    include:
+    - master
+  tags:
+    include:
+    - v*
+pr:
+  branches:
+    include:
+    - "*"
+
+# This pulls in a variable group from Azure. Variables can be encrypted or not.
+variables:
+- group: deploy-secrets
+
+jobs:
+- template: build-pipeline.yml@templates
+  parameters:
+    has_unit_tests: false
```

Comment:
Does this have a different behaviour that doing: `skip_tests: true`?

Reply:
It doesn't. But moving forward I hope we can move to another pair of settings called has_unit_tests (defaults to True) and has_ui_tests (defaults to False).

Otherwise we're using different terminologies.

Reply:
I introduced these two parameters when working with the UI automation with Francis.